src/theme/defaultTheme/radio.ts

```diff
@@ -0,0 +1,9 @@
+export const radio = {
+  base: "lib:flex lib:items-center lib:gap-2",
+  size: {
+    sm: "lib:text-xs",
+    md: "lib:text-sm",
+    lg: "lib:text-base",
+  },
```

Comment:
I dont think this one follows the design.

width, height of the radio differs & the text sizes remains same.